tests/torch/sparsity/movement/test_training_with_third_party.py

```diff
@@ -0,0 +1,38 @@
+from pathlib import Path
+import pytest
+import os
+
+from tests.common.helpers import PROJECT_ROOT
+from tests.common.helpers import TEST_ROOT
+from tests.torch.helpers import Command
+from tests.torch.test_sanity_third_party import create_command_line
+
+
+class TestMovementWithTransformers:
+    @pytest.fixture(autouse=True)
+    def setup(self, temp_folder):
+        self.VENV_PATH = str(temp_folder["venv"])
+        self.VENV_ACTIVATE = str(". {}/bin/activate".format(self.VENV_PATH))
+        self.PYTHON_EXECUTABLE = str("{}/bin/python".format(self.VENV_PATH))
+        self.TRANSFORMERS_REPO_PATH = str(os.path.join(temp_folder["repo"], "transformers"))
+        self.CUDA_VISIBLE_STRING = "export CUDA_VISIBLE_DEVICES=0;"
+        self.PATH_TO_PATCH = str(os.path.join(PROJECT_ROOT, "third_party_integration", "huggingface_transformers",
+                                              "0001-Modifications-for-NNCF-usage.patch"))
+
+    @pytest.mark.dependency(
+        depends=['test_sanity_third_party.py::TestTransformers::install_trans'], # TODO:(yujie): how to do cross-file dependency?
```

Comment:
I'd not do a dependency with the function from another class.
In fact, the installation should be a bit different (+ another patch for JPQD).

There are at least 2 possible options:
1) simply extract the common parts to a function and reuse it in both classes.
2) introduce a class that is responsible for environment setup. It can store all required paths (e.g. VENV_PATH, VENV_ACTIVATE...)
You could split `install_trans` into meaningful steps (e.g. install nncf, install hf, apply patch). 
For JPQD you just need to inherit this class and add one extra step - apply patch with JPQD-related application code.

Reply:
Thanks for instructions. I'll take a look later. 
As mentioned earlier, probably there will be no JPQD-related patch to transformers. So I  think for now i'll only use the same venv by `install_trans `